contracts/DefaultRegistryAccessManager.sol

```diff
@@ -7,25 +7,13 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 contract DefaultRegistryAccessManager is RegistryAccessManager {
     using SafeMath for uint256;
 
-    string public constant WRITE_PERMISSION = "canWriteTo-";
+    bytes32 public constant WRITE_PERMISSION = keccak256("canWriteTo-");
 
     // Allows a write if either a) the writer is that Registry's owner, or
     // b) the writer is writing to attribute foo and that writer already has
     // the canWriteTo-foo attribute set (in that same Registry)
-    function confirmWrite(address /*_who*/, string _attribute, uint256 /*_value*/, bytes32 /*_notes*/, address _admin) public returns (bool) {
+    function confirmWrite(address /*_who*/, bytes32 _attribute, uint256 /*_value*/, bytes32 /*_notes*/, address _admin) public returns (bool) {
         Registry client = Registry(msg.sender);
-        return (_admin == client.owner() || client.hasAttribute(_admin, strConcat(WRITE_PERMISSION, _attribute)));
+        return (_admin == client.owner() || client.hasAttribute(_admin, WRITE_PERMISSION ^ _attribute));
```

Comment:
I think this is safe for only one permission of this type. If there were a second permission, it would be unsafe to xor.

Reply:
We would have to make sure permissions we allow in this registry are not the same as any existing ones.

Reply:
For each proposed attribute we have to check `_attribute` and `WRITE_PERMISSION ^ _attribute`.